web/client/components/data/download/DownloadDialog.jsx

```diff
@@ -20,10 +20,13 @@ class DownloadDialog extends React.Component {
         onClose: PropTypes.func,
         onExport: PropTypes.func,
         onDownloadOptionChange: PropTypes.func,
+        onFormatOptionsFetch: PropTypes.func,
         downloadOptions: PropTypes.object,
         formats: PropTypes.array,
         srsList: PropTypes.array,
-        defaultSrs: PropTypes.string
+        defaultSrs: PropTypes.string,
+        layer: PropTypes.object,
+        formatsLoading: PropTypes.bool
```

Comment:
Can we add an optional filter to limit the available formats if we don't want to expose all of them? It should be configurable from the plugin .

Reply:
Config Example:
```
            {
              "name": "WFSDownload",
              "cfg": {
                "formatFilter":  {"SHAPE-ZIP": "Shapefile"}
              }
            }
```

Reply:
please add documentation for these configuration options and how to use them in the plugin's doc.

Reply:
Indicate also if there are breaking changes in configuration (in the P.R.) . E.g. if you configured some formats, you have to change your configurations this way.